transportable-udfs-codegen/src/main/java/com/linkedin/transport/codegen/HiveWrapperGenerator.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.codegen;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Multimap;
+import com.linkedin.transport.api.udf.StdUDF;
+import com.linkedin.transport.api.udf.TopLevelStdUDF;
+import com.squareup.javapoet.ClassName;
+import com.squareup.javapoet.JavaFile;
+import com.squareup.javapoet.MethodSpec;
+import com.squareup.javapoet.ParameterizedTypeName;
+import com.squareup.javapoet.TypeSpec;
+import com.squareup.javapoet.WildcardTypeName;
+import java.io.File;
+import java.util.Collection;
+import java.util.List;
+import javax.lang.model.element.Modifier;
+
+
+public class HiveWrapperGenerator implements WrapperGenerator {
```

Comment:
javadoc all classes and non-private methods. (even package private ones to help contributors)